container.go

```diff
@@ -326,6 +340,20 @@ func (container *container) Statistics() (Statistics, error) {
 	return properties.Statistics, nil
 }
 
+// ProcessList returns an array of ProcessListItems for the container
+func (container *container) ProcessList() ([]ProcessListItem, error) {
+	operation := "ProcessList"
+	title := "HCSShim::Container::" + operation
+	logrus.Debugf(title+" id=%s", container.id)
+	properties, err := container.properties(processListQuery)
+	if err != nil {
+		return []ProcessListItem{}, makeContainerError(container, operation, "", err)
```

Comment:
Good point. It can here. Wasn't thinking copy/pasting the statistics code :innocent"